src/utils/fs.rs

```diff
@@ -19,6 +19,25 @@ pub fn dir_is_empty(dir_path: &Path) -> bool {
     dir_path.read_dir().map(is_empty).unwrap_or_default()
 }
 
+/// Remove `path` asking the user to overwrite if necessary
+/// `Ok(Some(())` means the path is clear,
+/// `Ok(None)` means the user doesn't want to overwrite
+/// `Err(_)` is an error
+// ToDo: Actual type to translate the above might be clearer?
```

Comment:
What if you change:
```
Some() -> true
None -> false
```

Using `Result<bool>`

And add to the docs that "Returns whether the path is clear or not"

Reply:
Oh yeah, not sure why I didn't think of that.